src/database/sqlite/schema.rs

```diff
@@ -74,7 +74,8 @@ CREATE TABLE addresses (
 CREATE TABLE spend_transactions (
     id INTEGER PRIMARY KEY NOT NULL,
     psbt BLOB UNIQUE NOT NULL,
-    txid BLOB UNIQUE NOT NULL
+    txid BLOB UNIQUE NOT NULL,
+    updated_at INTEGER
```

Comment:
What do you think to make it NOT NULL, and populate the db with current time in case of preexisting spend_transactions ?

Reply:
I thought about doing this, but my thinking is that setting it to the current time (or `0` for that matter) is arbitrary and as meaningless as just being explicit that it was not set and going for `NULL`.

In addition, setting `NOT NULL` here would make the migration much more involved since we wouldn't be able to use `ALTER TABLE` (we'd have to create a new table, copy the content over, delete the old one and rename the new into the old).

Finally, this is only going to affect a tiny number of users anyway. (Those that will ever have used Liana <=0.3 and still have spend drafts from this time.)